www/scripts/CHABBLE.YouTubeClient.js

```diff
@@ -0,0 +1,93 @@
+﻿var CHABBLE = CHABBLE || {};
+
+CHABBLE.YouTubeClient = (function () {
+    var youTubePlayer;
+    var playState;
+    var readyCallback;
+    var finishedCallback;
+    var readyToPlay;
+
+    function playYouTubeVideo(videoId, startTime) {
+        var currentTime = youTubePlayer.getCurrentTime();
+        var outBy = Math.abs(currentTime - startTime);
+
+        if ( (playState === 1 && outBy < 10) || playState === 3)
```

Comment:
Not essential but could we make the play state stuff a bit easier to read by defining an object with the values mapped on to something readable at the start. so something like:
```js
var playState;
var playStates = {
   "not-started": 0
   "playing": 1,
   "paused": 3
};
```

This if statement could then read:
```js
if ( (playState === playStates.playing && outBy < 10) || playState === playStates.paused)
```

n.b. I don't know what the states actually are so the above is an example I made up randomly.